waiter/README.md

```diff
@@ -58,6 +58,10 @@ $ lein test :integration-slow
 $ WAITER_TEST_REQUEST_LATENCY_MAX_INSTANCES=30 APACHE_BENCH_DIR=/usr/bin lein test :perf
 ```
 
+## Code Walkthrough
+
+For more information on how Waiter is set up, check out this [code walkthrough](https://usecodeflow.com/tutorials/view/twosigma/waiter/tree/cb1000/ck8diy2g500000ul4s0b6l4zb).
```

Comment:
Walkthrough looks neat.

Does this link point to an immutable resource? That is are we sure the content isn't going to change?

Reply:
Also, can we monospace fonts for code snippets?

Reply:
Yup, immutable. I'll make you the owner of the walkthrough so only you can change it (when you login at https://usecodeflow.com/). Updated the fonts!